.github/workflows/reusable-build-on-alpine-static.yml

```diff
@@ -8,7 +8,8 @@ on:
         required: true
       release:
         type: boolean
-
+permissions:
```

Comment:
All jobs in this workflow need the `contents: write` permissions. Why do we need this?